components/DataCard.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from "react";
+import { View, Image, TouchableOpacity } from 'react-native';
+import { Text } from "native-base";
+
+// export default class WordSearch extends Component {
+
+export default DataCard = (props) => {
+  // render (){
+  const { image, word, _id } = props;
+  // console.log(props);
+  return (
+    <View>
+      <TouchableOpacity onPress={()=> props.recommendationTab()}>
+      <Image source={{ uri: image }} resizeMode='contain' resizeMethod='scale' style={props.style} onPress={()=> props.renderRec(word)} />
```

Comment:
Try removing the onPress from TouchableOpacity and replaceing the one that's on the image. If that works the we can deal with the renderRec thing next.